README.md

```diff
@@ -1,6 +1,33 @@
 # OpenSkos-API
 Restful API for OpenSkos
 
+## Development
+
+### Installation:
+
+* `composer install`
+* `vendor/bin/grumphp git:init` -- grump will make sure that you have no mistakes before commit ;)
```

Comment:
The git hooks should be installed during composer install already, perhaps we can leave this out.

Reply:
Yeah, you're right. I can remove it. or just give short explanation about how to disable/enable grump git-commit sniffer.